src/Selenium2Library/__init__.py

```diff
@@ -135,7 +135,13 @@ class Selenium2Library(
     ROBOT_LIBRARY_SCOPE = 'GLOBAL'
     ROBOT_LIBRARY_VERSION = VERSION
 
-    def __init__(self, timeout=5.0, implicit_wait=0.0, run_on_failure='Capture Page Screenshot'):
+    def __init__(self,
+                 timeout=5.0,
+                 implicit_wait=0.0,
+                 run_on_failure='Capture Page Screenshot',
+                 screenshot_root_directory=None
+    ):
```

Comment:
In Robot's own libs we have recently switched to use double backticks to get "code style" formatting. This is what we also recommend in Libdoc's docs:
http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#representing-arguments

We used single backticks earlier but nowadays we prefer to reserve it for internal linking. Code style looks better (that's obviously subjective) and avoids accidentally creating a link.

Hmm, should I add a note about this to Robot's newly added contribution guidelines? robotframework/robotframework#1805